microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -83,19 +82,69 @@ The capability to back up and restore the {product-title} database is now availa
 //[id="microshift-4-14-load-balancer"]
 //==== Deploying network load balancers on {product-title}
 
-//[id="microshift-4-14-storage"]
-//=== Storage
+[id="microshift-4-14-storage"]
+=== Storage
+
+[id="microshift-backup-and-restore"]
+=== Backup and restore
+The capability to back up and restore the {product-title} database is now available. You can manually back up and restore data on all supported systems at any time. See the xref:../microshift_backup_and_restore/microshift-backup-and-restore.adoc#microshift-backup-and-restore[Backing up and restoring {product-title} data] documentation for more information.
+
+//[id="microshift-4-14-running-apps"]
+//=== Running Applications
+//still need to add
 
 //[id="microshift-4-14-lvms-system-requirements"]
 //==== LVMS system requirements
 
-//[id="microshift-4-14-deprecated-removed-features"]
-//== Deprecated and removed features
+[id="microshift-4-14-deprecated-and-removed"]
+== Deprecated and removed features
 
-//[id="microshift-4-14-bug-fixes"]
-//== Bug fixes
+Some features available in previous releases of {product-title} have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the tables below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the following tables, features are marked with the following statuses:
+
+* _Available_
+
+* _Deprecated_
+
+* _Removed_
+
+.{product-title} deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 | 4.14
+
+|Network configuration flags
+|-
+|Depricated
+|Removed
+
+|CIDR notation
+|-
+|-
+|Removed
+
+|====
+[discrete]
+[id="microshift-4-14-network-config-flags-removed"]
+==== Network configuration flags are removed
 
-//etc
+The gateway bridge flag, 'gatewayInterface', and the OVS flag, `disableOVSInit`, in the networking configuration file, `/etc/microshift/ovn.yaml`, are removed with this release. See the xref:../microshift_networking/microshift-networking.adoc#microshift-config-OVN-K_microshift-networking[MicroShift OVN-K configuration] documentation for more information.
```

Comment:
@ShaunaDiaz is there a reroute for this link? It is breaking the build and I am not seeing where this file contents went.

Reply:
[id="microshift-applying-networking-settings"] for the assembly name and [id="microshift-config-OVN-K_{context}"] for the module in the update

Reply:
Thanks! Just wanted to double check